modules/nw-sriov-configuring-operator.adoc

```diff
@@ -103,45 +95,52 @@ Follow this procedure to disable it.
 * You must have installed the SR-IOV Operator.
 
 .Procedure
-. Update the default SriovOperatorConfig Custom Resource
+
+. To update the `default` SriovOperatorConfig CR, enter the following command:
 +
 ----
-$ oc patch sriovoperatorconfig default --patch '{ "spec": { "enableOperatorWebhook": false}}' --type=merge -n openshift-sriov-network-operator
+$ oc patch sriovoperatorconfig default --type=merge \
+  -n openshift-sriov-network-operator \
+  --patch '{ "spec": { "enableOperatorWebhook": false } }'
 ----
 
-. To re-enable Operator Admission Controller Webhook
+. To reenable the Operator Admission Controller Webhook, enter the following command:
 +
 ----
-$ oc patch sriovoperatorconfig default --patch '{ "spec": { "enableOperatorWebhook": true}}' --type=merge -n openshift-sriov-network-operator
+$ oc patch sriovoperatorconfig default --type=merge \
+  -n openshift-sriov-network-operator \
+  --patch '{ "spec": { "enableOperatorWebhook": true } }'
 ----
 
 [id="configuring-custom-nodeselector_{context}"]
-== Configuring Custom NodeSelector for SRIOV Network Config Daemon
+== Configuring a custom NodeSelector for SR-IOV Network Config Daemon
 
-The SR-IOV Network Config Daemon is in charge of discovering and configuring
-SRIOV network devices on the nodes.
+The SR-IOV Network Config Daemon discovers and configures the SR-IOV network devices on cluster nodes. By default, it will be deployed to all the `worker` nodes in the cluster. You can use node labels to specify on which nodes the SR-IOV Network Config Daemon runs.
 
-By default, it will be deployed to all the `worker` nodes in the cluster. You
-can follow this procedure to select the nodes, where the SR-IOV Network Config
-Daemon shall be deployed to, with node labels.
+To specify the nodes where the SR-IOV Network Config Daemon is deployed, complete the following procedure.
+
+[IMPORTANT]
+=====
+When you update the `configDaemonNodeSelector` parameter, the  SR-IOV Network Config Daemon is recreated on each selected node.
+While the daemon is recreated, cluster users will be unable to apply any new SR-IOV Network Node Policy or create new SR-IOV pods.
+=====
 
 .Procedure
-. Update the default SriovOperatorConfig Custom Resource
+
+. To update the `default` SriovOperatorConfig CR, enter the following command:
 +
 ----
-$ oc patch sriovoperatorconfig default --patch '[{"op": "replace", "path": "/spec/configDaemonNodeSelector", "value": {<node-label>}}]' --type=json -n openshift-sriov-network-operator
+$ oc patch sriovoperatorconfig default --type=json \
+  -n openshift-sriov-network-operator \
+  --patch '[{
+      "op": "replace",
+      "path": "/spec/configDaemonNodeSelector",
+      "value": {<node-label>}
```

Comment:
@zshi-redhat, does `value` expand out to

```
"value": {
 node-role.kubernetes.io/worker: "",
  some-other-key/worker: "",
  ...
}
```

Or can there be only 1 key provided? Or is this a different data structure than I think it is?

Thanks.